mirroroperator/registrymirror.py

```diff
@@ -336,24 +357,27 @@ def generate_stateful_set(self, stateful_set):
         pod_labels = {'component': 'registry'}
         pod_labels.update(self.labels)
         volumes = []
-        if self.cert_name:
+        if self.ca_certificate_bundle:
             volumes = [
                 client.V1Volume(
-                    name=self.cert_name,
+                    name=self.ca_certificate_bundle,
                     config_map=client.V1ConfigMapVolumeSource(
-                        name=self.cert_name
+                        name=self.ca_certificate_bundle
                     )
                 )
             ]
-        if self.secret_name:
+        if self.docker_certificate_secret:
             volumes.append(
                 client.V1Volume(
                     name="tls",
                     secret=client.V1SecretVolumeSource(
-                        secret_name=self.secret_name
+                        secret_name=self.docker_certificate_secret
                     ),
                 )
             )
+        else:
+            raise NameError('No docker certificate secret specified')
```

Comment:
This exception shouldn't be raised here, we should be breaking if this isn't provided on startup

Reply:
I added the docker_certificate_secret as a positional argument in the __init__ for RegistryMirror trying to fail on initialization if it's not provided. How should we break before that?